streaming/src/main/scala/org/apache/spark/streaming/scheduler/JobScheduler.scala

```diff
@@ -93,6 +103,10 @@ class JobScheduler(val ssc: StreamingContext) extends Logging {
       receiverTracker.stop(processAllReceivedData)
     }
 
+    if (executorAllocationManager != null) {
```

Comment:
I agree with Holden. I don't think it's possible for it to be null. The only chance that could happen is if you call `stop()` in the constructor before declaring all the variables, which is highly improbable.